tests/customer_factory.py

```diff
@@ -14,4 +15,20 @@ class Meta:  # pylint: disable=too-few-public-methods
     first_name = factory.Faker("first_name")
     last_name = factory.Faker("last_name")
     email = factory.Faker("email")
-    password = factory.Faker("password")
\ No newline at end of file
+    password = factory.Faker("password")
+
+class AddressFactory(factory.Factory):
+    """ Creates fake addresses that you don't have to support """
+    class Meta:
+        """Maps factory to data model"""
+
+        model = Address
+
+    address_id = factory.Sequence(lambda n: n)
```

Comment:
address_id might require change to just "id"  here too.

Reply:
Having "address_id" here instead of having "id" to remove the confusion with "id" present in the customer which denotes customer id, hope that clears it.